### DISCUSSION TITLE
using with Suspense for data fetching

### DISCUSSION THREAD

**Discussion by dante01yoon** [facebookexperimental/Recoil#2093] [Q&A]
Hi, I'm new to Recoil and wonder there is no problem using with Suspense in React 18. 
Is using Suspense with Recoil is currently experimetal feature like [using Suspense with react-query](https://tanstack.com/query/v4/docs/guides/suspense)?

**Comment by drarmstr** [comment]
Suspense is supported and an integral part of how Recoil supports async states.
